### DISCUSSION TITLE
Latest Version Error

### DISCUSSION THREAD

**Discussion by BPowell76** [requarks/wiki#7371] [Error / Bug Report] [answered]
In the Admin > System Info pane, the current value of "Latest Version" is 2.5.303. The update notification won't fire. After updating to version 2.5.304, the latest version is still showing 2.5.303 while the installed version is showing 2.5.304.

![image](https://github.com/user-attachments/assets/50232731-8163-4047-a15b-1c99cada5965)

I've been trying to find the value in the development version of Wiki.js, but no luck so far.

**Comment by NGPixel** [comment, marked as answer]
Fixed. It might take a day before it shows correctly in your installation though.
